src/application/tally/widgets/organization-proposals.container.tsx

```diff
@@ -30,17 +24,9 @@ export const OrganizationProposalsContainer = ({
     null,
   );
 
-  const proposalQuery = useCommandQuery({
-    command: new GetTallyProposalsCommand({
-      twitterHandle: userHandle,
-      afterCursor: currentProposalCursor,
-    }),
-    retry: 5,
-    retryDelay: 1800,
-    staleTime: Number.POSITIVE_INFINITY,
-    placeholderData: (previousData) => {
-      return previousData;
-    },
+  const proposalQuery = useProposalsQuery({
+    username,
+    afterCursor: currentProposalCursor,
```

Comment:
as above, the isPlaceholderData check can be removed from the isLoadingProposal variable